modules/nf-scil/betcrop/fslbetcrop/main.nf

```diff
@@ -23,31 +23,49 @@ process BETCROP_FSLBETCROP {
     def prefix = task.ext.prefix ?: "${meta.id}"
 
     def b0_thr = task.ext.b0_thr ? "--b0_thr " + task.ext.b0_thr : ""
-    def bet_dwi_f = task.ext.bet_dwi_f ? "-f " + task.ext.bet_dwi_f : ""
+    def bet_f = task.ext.bet_f ? "-f " + task.ext.bet_f : ""
+    def size_dil = task.ext.size_dil ? task.ext.size_dil : ""
 
     """
     export ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS=1
     export OMP_NUM_THREADS=1
     export OPENBLAS_NUM_THREADS=1
 
-    scil_extract_b0.py $dwi $bval $bvec ${prefix}__b0.nii.gz --mean \
-        $b0_thr --force_b0_threshold
-    bet ${prefix}__b0.nii.gz ${prefix}__b0_bet.nii.gz -m -R $bet_dwi_f
-    scil_image_math.py convert ${prefix}__b0_bet_mask.nii.gz ${prefix}__b0_bet_mask.nii.gz --data_type uint8 -f
-    mrcalc $dwi ${prefix}__b0_bet_mask.nii.gz -mult ${prefix}__dwi_bet.nii.gz -quiet -nthreads 1
+    if [[ -f "$bval" ]]
+    then
+        scil_extract_b0.py $image $bval $bvec ${prefix}__b0.nii.gz --mean \
+            $b0_thr --force_b0_threshold
 
-    scil_crop_volume.py $dwi ${prefix}__dwi_bet_cropped.nii.gz -f \
-        --output_bbox ${prefix}__dwi_boundingBox.pkl -f
-    scil_crop_volume.py ${prefix}__b0_bet_mask.nii.gz ${prefix}__dwi_bet_cropped_mask.nii.gz -f\
-        --input_bbox ${prefix}__dwi_boundingBox.pkl -f
-    scil_image_math.py convert ${prefix}__dwi_bet_cropped_mask.nii.gz ${prefix}__dwi_bet_cropped_mask.nii.gz \
-        --data_type uint8 -f
+        bet ${prefix}__b0.nii.gz ${prefix}__b0_bet.nii.gz -m -R $bet_f
+        scil_image_math.py convert ${prefix}__b0_bet_mask.nii.gz ${prefix}__image_bet_mask.nii.gz --data_type uint8 -f
+        mrcalc $image ${prefix}__b0_bet_mask.nii.gz -mult ${prefix}__image_bet.nii.gz -quiet -nthreads 1
```

Comment:
Is the use here of `${prefix}__b0_bet_mask.nii.gz` rather than `${prefix}__image_bet_mask.nii.gz` intended ? It shouldn't change anything on the output, but just validating!

Reply:
the use of ${prefix}__image_bet_mask.nii.gz is on purpose, because I need to use a common name for every mask when i want to use dilate or crop option. Otherwise if i use something like ${prefix}__*_bet_mask.nii.gz it can't distinguish between the different masks and the command broke.